### DISCUSSION TITLE
Error, system call for...

### DISCUSSION THREAD

**Discussion by Sunshaoe** [ablab/spades#1280] [Q&A]
HrrOr三-/home/sunshaoe/c9219/09219/corrected/configs/configLinux/bin/spades-hammer'info'" finished abnormally, Os return value: 1

[params.txt](https://github.com/ablab/spades/files/15048610/params.txt)
[spades.log](https://github.com/ablab/spades/files/15048611/spades.log)

**Comment by asl** [comment]
The log clearly reads:
```
  0:58:55.222     1M / 19G   ERROR   General                 (mmapped_reader.hpp        :  52)   mmap(2) failed. Reason: Cannot allocate memory. Error code: 12
```

So you are running out of RAM
